documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+       1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: groups of competitions may have modified nicknames and IDs to ensure consistency with past and future competitions.
+2. Location(s)
+   1. A location consists of the country, city, and venue.
+   2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
+   3. Competitions will be accepted if there are no other competitions within 100 km driving distance and less than 19 days away.
+      1. Otherwise, the WCA Board should review the circumstances of the proposed competition. A Delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Possible arguments that support such a competition being accepted include:
+         1. The two competitions don’t have events in common.
+         2. The two competitions take place in different countries.
+   4. 3x3x3 Fewest Moves simultaneous competitions
+      1. 3x3x3 Fewest Moves simultaneous competitions are WCA competitions having only the 3x3x3 Fewest Moves event and taking place simultaneously in different locations.
+         1. If any location of such competitions is closer than 100 km to any other competition, then
+            1. both competitions should be at least 5 days apart, or
+            2. the other competition must not have the 3x3x3 Fewest Moves event.
+      2. A 3x3x3 Fewest Moves simultaneous competition cannot be part of any other regular competition.
+         1. Time frames when both are held cannot overlap.
+   5. Twin competitions
+      1. Twin competitions are WCA Competitions that are held less than 5 days apart for which the distance between them is less than 100 km, with the intention to accommodate more competitors to compete in one area.
+         1. Twin competitions may be accepted if the need for competitions in an area is so great that the community will benefit from having multiple competitions held in a short period of time.
+      2. Twin competitions are obliged to clearly and publicly state their condition of twin competitions and their special participation terms.
+         1. A competitor is only permitted to participate in one of the twin competitions.
+            1. If a competitor competes at more than one of the involved twin competitions, only the results from the competition that they compete in first will be kept.
+3. WCA Delegate(s)
+4. Organizer(s)
+   1. Organizers can be specific persons and/or organizations.
+5. Competitor limit
+   1. The competitor limit can be on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date.
+      1. Qualification for an event must be based only on results from the same event.
```

Comment:
As @Jambrose777 suggested [in my suggestion about Z4+](https://github.com/thewca/wca-regulations/issues/719) it would be good to have that clarity in this document, since the optional regulations will live here. 

Maybe we could have something like:
```suggestion
      1. Qualification for an event must be based only on results from the same event.
      2. Qualification for an event should be based on one type of result (e.g. average/mean or single) with the second type of result used only to distinguish between competitors with an identical position using the preferred result.
```
Wording of the second part is quite clunky however - happy for someone to suggest something better! The core idea is that qualification results should be clear from the start (so people know what they need to achieve) and without ambiguity (e.g. it should be clear if it is average or single).

Reply:
```suggestion
      1. Qualification for an event must be based only on results from the same event.
      2. Qualification for an event must be based on one type of result only (i.e. average/mean or single)
         1. Exception: in case of using average/mean results, single results must be used to undo ties.
```

Does this look clearer?

Reply:
This looks good @AlbertoPdRF - I think having it as a sub-point is definitely the way to go. Do we want it to say "undo ties" or "break ties"?

Reply:
Probably "break ties" sounds better? I'll let a native speaker reply to that question :stuck_out_tongue:

Reply:
Break ties is correct.

Reply:
Changed to `break ties`, I'm committing this!